chat-widget/src/components/livechatwidget/common/reconnectChatHelper.ts

```diff
@@ -107,10 +107,12 @@ const redirectPage = (redirectURL: string, redirectInSameWindow: boolean) => {
     }
 };
 
-const isReconnectEnabled = (chatConfig: ChatConfig) => {
-    return chatConfig &&
-        chatConfig.LiveWSAndLiveChatEngJoin?.msdyn_enablechatreconnect &&
-        (chatConfig.LiveWSAndLiveChatEngJoin.msdyn_enablechatreconnect === "true" || chatConfig.LiveWSAndLiveChatEngJoin.msdyn_enablechatreconnect === true);
+const isReconnectEnabled = (chatConfig?: ChatConfig): boolean => {
+    if (chatConfig) {
+        const reconnectEnabled = (chatConfig.LiveWSAndLiveChatEngJoin?.msdyn_enablechatreconnect?.toLowerCase() === "true");
+        return reconnectEnabled;
```

Comment:
nit: return chatConfig.LiveWSAndLiveChatEngJoin?.msdyn_enablechatreconnect?.toLowerCase() === "true";

Reply:
This is intentional due to the complexity of .(dot)s here for improve readability of code.